scapy/layers/http.py

```diff
@@ -616,12 +619,18 @@ def http_request(host, path="/", port=80, timeout=3,
     }
     http_headers.update(headers)
     req = HTTP() / HTTPRequest(**http_headers)
-    tcp_client = TCP_client.tcplink(HTTP, host, 80)
+    tcp_client = TCP_client.tcplink(HTTP, host, 80, debug=verbose)
     ans = None
+    if iptables:
+        ip = tcp_client.atmt.dst
+        iptables_rule = "iptables -%c INPUT -s %s -p tcp --sport 80 -j DROP"
```

Comment:
Could you add `--tcp-flags ALL RST` ?

Reply:
Also, I think that this should be `OUTPUT` instead of ` INPUT`

Reply:
I copy pasted the rule you made in the HTTP tests.
When I try using output *or* tcp flags, it fails..

Reply:
What is the goal of this rule? Drop everything or only RST segments?